src/code/z_sub_s.c

```diff
@@ -987,4 +1110,51 @@ s32 func_8013E8F8(Actor* actor, GlobalContext* globalCtx, f32 xzRange, f32 yRang
     return func_8013E748(actor, globalCtx, xzRange, yRange, exhangeItemId, &yawTols, SubS_ActorAndPlayerFaceEachOther);
 }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013E950.s")
+/**
+ * Computes the necessary HeadRot and TorsoRot adjustments to smoothly turn an actors's head and torso to different yaw
+ * and pitch target points
+ *
+ * @param[in] worldPos the actor's world position
+ * @param[in] focusPos the actor's focus position
+ * @param[in] shapeYRot the actor's shape's Y rotation
+ * @param[in] yawTarget the target point to determine desired yaw
+ * @param[in] pitchTarget the target point to determine desired pitch
+ * @param[in,out] headZRotAdj the computed actors' head's Z rotation
+ * @param[in,out] headXRotAdj the computed actors' head's X rotation
+ * @param[in,out] torsoZRotAdj the computed actors' torso's Z rotation
+ * @param[in,out] torsoXRotAdj the computed actors' torso's X rotation
+ * @param[in] headZRotAdjMax the max head's Z rotation
+ * @param[in] headXRotAdjMax the max head's X rotation
+ * @param[in] torsoZRotAdjMax the max torso's Z rotation
+ * @param[in] torsoXRotAdjMax the max torso's X rotation
+ */
+s32 SubS_TurnToPointMultiTarget(Vec3f* worldPos, Vec3f* focusPos, s16 shapeYRot, Vec3f* yawTarget, Vec3f* pitchTarget,
```

Comment:
So looking again at this function and its difference between the other `TurnToPoint` above, I think the more important difference is that this calculates the rotation adjustment to add to the normal rotation, rather than the full rotation. With this in mind I think `SubS_TurnToPointAdj` would be a better name and will probably change it to that unless someone has objections.

Reply:
Is there a reason to use `adj` over `step`? That's the only thing I'm puzzled about here.

Reply:
Not really. I'm mainly using `adj` just because that is the affix in `EnBji01`'s struct, so didn't even think of `step`.

Reply:
I'm thinking analogously to the z_lib functions. Of course now we know those are called things like `chase`...

Reply:
Ok, I think i'll go with `SubS_TurnToPointStep` instead. I'll also change the arg names. What about `EnBji01`'s struct, should those names also be changed?

Reply:
Yes, at least for consistency.